### PR TITLE
vendor: add gofail/runtime

### PR DESCRIPTION
**Pull request by gyuho**
to support `FAILPOINTS=1 ./build` with vendoring

### CODE REVIEW

File: scripts/updatedep.sh [hunk 1]
```diff
@@ -36,6 +36,10 @@ pushd "${GLIDE_ROOT}"
 	go install
 popd
 
+echo "vendoring gofail/runtime"
```
Comment (heyitsanthony) [inline scripts/updatedep.sh]: is this necessary? the updatedep script probably shouldn't have special rules for any deps

**Comment by gyuho** [comment]
@heyitsanthony You are right. Just fixed. PTAL. Thanks.

**Comment by gyuho** [comment]
Anthony figured a way to fix `gopath` in build script. Closing. We can vendor later when we need `gofail` enabled tests in CI.

**Pull request closed without merging**